### ISSUE TITLE
Support class_weights to loss function

### ISSUE THREAD

**Issue by innat** [innat/medic-ai#78]
```python
loss = SparseDiceCELoss(
    num_classes=3,
    class_weights=[1.0, 1.0, 5.0], 
)

loss = SparseDiceCELoss(
    num_classes=3,
    class_weights={0:1.0, 1:1.0, 2:5.0}
)
```
